### ISSUE TITLE
Single-Duct System Auto Heat/Cool Setpoint Manager

### ISSUE THREAD

**Issue by aaron-boranian** [NREL/EnergyPlus#6658]
Problem: For single-duct rooftop systems (VAV, multispeed, and constant volume) with central heating and cooling coils, there is no way to automatically change setpoints to switch from heating to cooling.

Rationale: Users frequently build a VAV system and set the heating coil setpoint to a hot supply temp and set the cooling coil setpoint to a cold supply temp. This causes poor control and the coils will battle continually. Advanced single-zone rooftop units may have exotic features that require modeling as built-up air handlers because not all the features are available in the unitary system; these need to be controllable without resorting to EMS.

Solution: add a new setpoint manager that can coordinate the setpoints of a central heating and cooling coil to meet the needs of the zones without causing the coils to battle.

Context: It might be feasible to use SetpointManager:SingleZone:Reheat on a single-zone system to apply the same setpoint to both coils, but that’s not it’s intended use. SetpointManager:Warmest can effectively turn off the cooling coil when cooling is no longer needed, but SetpointManager:Coldest isn’t currently applicable to single duct systems.

Suggestion from legacy enhancement list (Control_2009_03).

Migrated from [UserVoice](http://energyplus.uservoice.com/forums/258860/suggestions/13660686) feedback from @EnergyArchmage 

**NOTE**: This may have been settled in PR #5976. Would @Nigusse or @mjwitte like to chime in?

**Comment by mjwitte** [comment]
Yes, #5976 "Allowed SetpointManager:Coldest to work with single-duct systems" did help with part of this. It still requires some though on the part of the user to make sure coils don't battle.

This is also related to #5217 "Eliminate use of single setpoint controlling heating and cooling coils at the same time". 

Also, this setpoint manager could do the fan heat adjustment and eliminate the need for SetpointManager;MixedAir. And set humidity control setpoints also. So, instead of requiring 3 or 4 different setpoint manager, this single manager could control everything.
